uwblueprint/project-read-backend
# Add POST user endpoint

### Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/POST-user-endpoint-17370b16be7c4b7d918840a70ccd75d2)



### Implementation description

-



### Steps to test

1. Make sure the user does not exist in firebase or local db
2. hit the `create` endpoint with a email in the body
3. check if the user exists in Firebase and in our local db with matching firebase UIDs

### What should reviewers focus on?

- Any edge cases where firebase/local failures can cause an unstable state for user management
